c_api/utils/utils_c.h

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// Copyright 2004-present Facebook. All Rights Reserved.
+// -*- c -*-
+
+#ifndef FAISS_UTILS_C_H
+#define FAISS_UTILS_C_H
+
+#include <stdint.h>
+#include <stdlib.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+const char* get_version();
```

Comment:
All functions in the C API have the `faiss_` prefix, otherwise they might collide with other functions (whereas the C++ library puts everything in namespaces).